docs/.vuepress/config.js

```diff
@@ -105,11 +121,41 @@ module.exports = {
       }
     },
     footer: {
-      logo: "/logo-bw.svg",
+      question: {
+        text: "Chat with IBC developers in <a href='https://discord.gg/W8trcGV' target='_blank'>Discord</a>."
+      },
       textLink: {
         text: "ibcprotocol.org",
         url: "https://ibcprotocol.org"
       },
+      services: [
+        {
+          service: "medium",
+          url: "https://blog.cosmos.network/"
+        },
+        {
+          service: "twitter",
+          url: "https://twitter.com/cosmos"
+        },
+        {
+          service: "linkedin",
+          url: "https://www.linkedin.com/company/tendermint/"
+        },
```

Comment:
This should be removed in my opinion or we should link the Interchain linkedin, but the company Tendermint has no ownership of ibc-go